config/releases.exs

```diff
@@ -15,6 +15,12 @@ config :help_desk,
        region: help_desk_config["SQS_QUEUE_REGION"]
      ]}
 
+config :appsignal, :config,
```

Comment:
Appsignal configuration was missing, so I'll add that to an infrastructure PR later on